deno/generate_short_url/src/mod.ts

```diff
@@ -0,0 +1,67 @@
+export default async function (req: any, res: any) {
+  const { provider, url } = JSON.parse(req.payload);
+
+  if (!provider) {
+    throw new Error("provider is required");
+  }
+
+  if (!url) {
+    throw new Error("url is required");
+  }
+
+  if (provider === "bitly") {
+    const BITLY_TOKEN = req.variables["BITLY_TOKEN"];
+
+    if (!BITLY_TOKEN) {
+      throw new Error("BITLY_TOKEN is required");
```

Comment:
Instead of throwing an error, please return a response as defined in the issue.

```suggestion
      res.json({ success: false, message: "BITLY_TOKEN is required"});
      return;
```

Please make sure to do this for the other errors too.